libraries/AP_BattMonitor/AP_BattMonitor_Backend.cpp

```diff
@@ -161,13 +161,21 @@ bool AP_BattMonitor_Backend::arming_checks(char * buffer, size_t buflen) const
                                 (_state.voltage < _params._arming_minimum_voltage);
     bool below_arming_capacity = (_params._arming_minimum_capacity > 0) &&
                                  ((_params._pack_capacity - _state.consumed_mah) < _params._arming_minimum_capacity);
+    bool fs_capacity_inversion = is_positive(_params._critical_capacity) &&
+                                 is_positive(_params._low_capacity) &&
+                                 (_params._low_capacity <= _params._critical_capacity);
+    bool fs_voltage_inversion = is_positive(_params._critical_voltage) &&
+                                is_positive(_params._low_voltage) &&
+                                (_params._low_voltage <= _params._critical_voltage);
 
     bool result = update_check(buflen, buffer, low_voltage,  "low voltage failsafe");
     result = result && update_check(buflen, buffer, low_capacity, "low capacity failsafe");
     result = result && update_check(buflen, buffer, critical_voltage, "critical voltage failsafe");
     result = result && update_check(buflen, buffer, critical_capacity, "critical capacity failsafe");
     result = result && update_check(buflen, buffer, below_arming_voltage, "below minimum arming voltage");
     result = result && update_check(buflen, buffer, below_arming_capacity, "below minimum arming capacity");
+    result = result && update_check(buflen, buffer, fs_capacity_inversion, "capacity failsafe critical > low");
+    result = result && update_check(buflen, buffer, fs_voltage_inversion, "voltage failsafe critical > low");
```

Comment:
This should be >= but I'm guessing we don't have space?

Reply:
We do on voltage, we don't on capacity. It was correct before I made it a <= at peters suggestion :(

Reply:
It's not crazy problematic, but we should fix the voltage one then. We could also replace failsafe with FS so it fits on both.

Reply:
I actually really don't like `FS`. It's not very userfriendly, and if English isn't your first language doesn't translate well at all. It generally just is out of place in user facing errors IMO.

Reply:
Yeah, not my preference at all and it is fine not doing it. I am worried that someone making it equal won't understand what is going on though - and the only way to know if looking at code.

I was initially going to comment that making it equal would be a strange but valid configuration, but then saw that you changed it on @peterbarker's suggestion.

Anyway, I'm not fussed on this, I think that we should fix it on voltage given we have space, but otherwise I'm good to go with this.

Reply:
Killed the = part of the check, because if a user configures it that way it's at least obvious what happens, the primary purpose of this is to catch someone getting confused in what order things trigger. If they set the triggers to the same level then it's easily understood what happens. (If silly)